### PR TITLE
more explanation and examples for setup-game.sh script

### PR DESCRIPTION
**Pull request by CarlinWilliamson**
We had _issues_ running a practise game today due to user error with this script...

Added an example usage and more hints about the syntax

### CODE REVIEW

**Comment by jayenashar** [comment]
can we add the examples to `setup-game.config.sh` as comments?

please note that the intention now is for the config is committed to git, to keep a record of practice and real games.

File: source/game_preparation/setup_game_script.rst [hunk 1]
```diff
@@ -9,6 +9,71 @@ The script is situated in ``$RUNSWIFT_CHECKOUT_DIR/bin/setup-game``, and can be
...
+    Warning: Permanently added 'gandalf.local' (ECDSA) to the list of known hosts.
+     [+] [ 0%] gandalf: Setting up on team 18 and player 1
+     [+] [ 0%] gandalf: Restarting wifi
+     [+] [100%] Setup complete
+    1/1 players synced: gandalf
+    Ready to play.
+
+.. note:: 
+    Robot names must be lower case.
+    To setup two teams enter space seperated values (no commas).
```
Comment (pzrq) [inline source/game_preparation/setup_game_script.rst]: Typo: Make it “separated”

File: source/game_preparation/setup_game_script.rst [hunk 2]
```diff
@@ -9,6 +9,71 @@ The script is situated in ``$RUNSWIFT_CHECKOUT_DIR/bin/setup-game``, and can be
...
+    
+    For example, two teams might be configures as such:
```
Comment (pzrq) [inline source/game_preparation/setup_game_script.rst]: Grammar: “configured”

**Review by pzrq — CHANGES_REQUESTED** [review]
Great start.

I think we can have this and also follow Jayen’s suggestion if time allows - others need not wait until code release for human-readable documentation updates.

**Comment by pzrq** [comment]
@abhivij If you have suggestions as you also learnt this with us today, we’d also be happy to hear them 😄

File: source/game_preparation/setup_game_script.rst (hunk 1 above)
Comment (CarlinWilliamson) [reply source/game_preparation/setup_game_script.rst]:
```suggestion
    To setup two teams enter space separated values (no commas).
```
I can't spell to save my life XD

File: source/game_preparation/setup_game_script.rst (hunk 2 above)
Comment (CarlinWilliamson) [reply source/game_preparation/setup_game_script.rst]:
```suggestion
    For example, two teams might be configured as such: 
```

**Comment by CarlinWilliamson** [comment]
> can we add the examples to `setup-game.config.sh` as comments?
> 
> please note that the intention now is for the config is committed to git, to keep a record of practice and real games.

We could but adding another example to that file will have `setup-game.sh` the actual and example which could lead to more confusion.
Another option would be to have a practise game setup as the default values but this could lead to some interesting accidents at competition.

**Comment by CarlinWilliamson** [comment]
> > can we add the examples to `setup-game.config.sh` as comments?
> > please note that the intention now is for the config is committed to git, to keep a record of practice and real games.
> 
> We could but adding another example to that file will have `setup-game.sh` the actual and example which could lead to more confusion.
> Another option would be to have a practise game setup as the default values but this could lead to some interesting accidents at competition.

Kindof poor excuse isn't it...
Here is a [PR](https://github.com/UNSWComputing/rUNSWift/pull/2400) which adds the second example to `setup-game.config.sh`

I suppose this PR should arguably be closed?

**Comment by jayenashar** [comment]
> We could but adding another example to that file will have `setup-game.sh` the actual and example which could lead to more confusion.

Leave it commented out

> Another option would be to have a practise game setup as the default values but this could lead to some interesting accidents at competition.

If you commit every practice/competition game, it's less of an issue.

**Review by pzrq — APPROVED** [review]
Looks good to me

I’d be inclined not to overthink it, different team members will have different communication preferences and styles and that’s generally to be celebrated.

**Comment by pzrq** [comment]
Happy to handle any further feedback in future PRs 😄

**Pull request merged**